docs/content/practices/emulator_vs_real_device.md

```diff
@@ -1,3 +1,176 @@
-# Emulator vs real device
+# Emulator vs Real device
 
-To be done
\ No newline at end of file
+This question is a trade off and there is no right and wrong answers. We'll review pros/cons and basic emulator setup on
+CI
+
+## Real device
+
+Here is pros/cons
+
+➕ Real environment </br>
+➕ Doesn't consume CI resources
+
+➖ Breaks often </br>
+➖ Works slow </br>
+➖ Requires special conditions </br>
+
+A real device will help you to catch more bugs from the first perspective, however talking about scalability, if you
+have a lot of devices, you need to locate them in a special room with no direct sunlight and with a climate-control.
+
+However, it doesn't save from disk and battery deterioration, because they are always charging and performs I/O
+operations. It may be a reason of your tests failing not because of them caught a real bug, but because of an issue with
+a device.
+
+## Emulator
+
+Here is pros/cons
+
+➕ Easy configurable </br>
+➕ Works faster </br>
```

Comment:
add reminder that it can be achieved trough specific configs just because it's not true in general